# Service to Set Joints
float64[] position

---
int64 ret
string msg